examples/gups_basic/main.pony

```diff
@@ -1,14 +1,44 @@
+use "cli"
 use "collections"
-use "options"
 use "time"
 
+class val Config
+  let logtable: USize
+  let iterate: USize
+  let chunk: USize
+  let streamer_count: USize
+  let updater_count: USize
+
+  new val create(env: Env) ? =>
+    let cs = CommandSpec.parent("gups_basic", "", [
+      OptionSpec.i64("table", "Log2 of the total table size." where default' = 20)
+      OptionSpec.i64("iterate", "Number of iterations." where default' = 10000)
+      OptionSpec.i64("chunk", "Chunk size." where default' = 1024)
+      OptionSpec.i64("streamers", "Number of streamers." where default' = 4)
+      OptionSpec.i64("updaters", "Number of updaters." where default' = 8)
+    ]).>add_help()
+    let cmd =
+      match CommandParser(cs).parse(env.args, env.vars())
+      | let c: Command => c
+      | let ch: CommandHelp =>
+          ch.print_help(OutWriter(env.out))
+          env.exitcode(0)
+          error
+      | let se: SyntaxError =>
+          env.out.print(se.string())
+          env.exitcode(1)
+          error
```

Comment:
Using extra indentation in `match` case bodies doesn't match the standard library style - these should be at the same indentation level as the `let`.

This comment also applies to some other `match` blocks in this PR.